test/integration/custom-routes/test/index.test.js

```diff
@@ -151,6 +152,26 @@ const runTests = (isDev = false) => {
     expect(await getBrowserBodyText(browser)).toMatch(/Hello again/)
   })
 
+  it('should match a page after a rewrite', async () => {
+    const html = await renderViaHTTP(appPort, '/to-hello')
+    expect(html).toContain('Hello')
+  })
+
+  it('should match dynamic route after rewrite', async () => {
+    const html = await renderViaHTTP(appPort, '/blog/post-1')
+    expect(html).toMatch(/post:.*?post-2/)
+  })
+
+  it('should match /_next file after rewrite', async () => {
```

Comment:
We need this same test for a public file that is on the same "level" as a dynamic route.

Reply:
Added a test for this case